src/main/java/frc/robot/subsystems/climb/Climb.java

```diff
@@ -0,0 +1,127 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+package frc.robot.subsystems.climb;
+
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
+import frc.robot.subsystems.gyro.GyroIONavX2;
+
+import org.littletonrobotics.junction.Logger;
+
+public class Climb extends SubsystemBase
+{
+    private final ClimbIO                 _io;
+    private final ClimbIOInputsAutoLogged _inputs = new ClimbIOInputsAutoLogged();
+    private final PIDController           _tiltPID;
+    private final PIDController           _leftPID;
+    private final PIDController           _rightPID;
+
+    // Average height that the two arms should be set to
+    private Double _desiredHeight;
+    private double _climbMinExtension = Constants.Climb.MIN_EXTENSION; // TODO: tune value
+    private double _climbMaxExtension = Constants.Climb.MAX_EXTENSION; // TODO: tune value
+    GyroIONavX2    _gyroIONavX2;
+
+    public Climb(ClimbIO io)
+    {
+        _io = io;
```

Comment:
We'll need to pass a reference to our `Gyro` subsystem in through the constructor and save it off to the member variable from above:

```suggestion
    public Climb(Gyro gyro, ClimbIO io)
    {
        _gyro = gyro;
        _io = io;
```